wpilibj/src/main/java/edu/wpi/first/wpilibj/DriverStation.java

```diff
@@ -89,6 +90,72 @@ public enum MatchType {
     Elimination
   }
 
+  /** A controller POV direction. */
+  public enum POVDirection {
+    /** POV center. */
+    Center(0x00),
+    /** POV up. */
+    Up(0x01),
+    /** POV up right. */
+    UpRight(0x01 | 0x02),
+    /** POV right. */
+    Right(0x02),
+    /** POV down right. */
+    DownRight(0x02 | 0x04),
+    /** POV down. */
+    Down(0x04),
+    /** POV down left. */
+    DownLeft(0x04 | 0x08),
+    /** POV left. */
+    Left(0x08),
+    /** POV up left. */
+    UpLeft(0x01 | 0x08);
+
+    /**
+     * Converts a byte value into a POVDirection enum value.
+     *
+     * @param value The byte value to convert.
+     * @return The corresponding POVDirection enum value.
+     * @throws IllegalArgumentException If value does not correspond to a POVDirection.
+     */
+    private static POVDirection of(byte value) {
+      for (var direction : values()) {
+        if (direction.value == value) {
+          return direction;
+        }
+      }
+      throw new IllegalArgumentException("Invalid value " + value + "!");
```

Comment:
We should make invalid values be center and print an error, rather than throwing. There have been controllers in the past that have allowed 3 buttons, and they could possibly crash. I'm not sure I know what SDL does and if I should trust it.